Classes/View/MjmlBasedView.php

```diff
@@ -3,12 +3,13 @@
 namespace Saccas\Mjml\View;
 
 use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Mvc\Cli\Command;
 use TYPO3\CMS\Fluid\View\StandaloneView;
 use TYPO3\CMS\Core\Utility\CommandUtility;
 
 class MjmlBasedView extends StandaloneView
 {
-    function render()
+    function render($actionName = null)
```

Comment:
`public` prefix is still missing, to match parent signature, see: https://github.com/TYPO3/Fluid/blob/master/src/View/AbstractTemplateView.php#L168